functests/5_latency_testing/latency_testing.go

```diff
@@ -267,3 +270,22 @@ func getNegativeTests(toolToTest string) []latencyTest {
 	}
 	return testSet
 }
+
+func getToolArgsRegex(tst latencyTest) []string {
+	//this lists of args depend on the ones the latency tool runners adds to tool command in cnf-features-deploy.
+	switch tst.toolToTest {
+	case oslat:
+		return []string{"--duration " + tst.testRuntime, "--rtprio ", "--cpu-list ", "--cpu-main-thread "}
+	case cyclictest:
+		return []string{"--duration " + tst.testRuntime, "--priority 95", "--threads ", "--affinity ", "--histogram ", "--interval ", "--mlockall ", "--quiet"}
+	case hwlatdetect:
+		thr := tst.testMaxLatency
+		if tst.hwlatdetectMaxLatency != "" {
+			thr = tst.hwlatdetectMaxLatency
+		}
+		return []string{"--duration " + tst.testRuntime, "--threshold " + thr, "--hardlimit " + thr, "--window ", "--width "}
+	default:
+		return nil
+	}
+	return nil
```

Comment:
this is probably the unreachable code that the prow-images lane is complaining about.

Reply:
thanks for pointing to that!